DoSomething/rogue
# Reviews table

#### What's this PR do?
- When a photo post is reviewed, a review is added to the reviews table. 
- Updates `Reviews` table to include `postable_id` and `postable_type`.
- Updates relations between reviews and posts.
- Updates documentation (now this endpoint requires a `reviewer` and `event_type`).

#### How should this be reviewed?
- Post a review to the `/reviews` endpoint. 
  - Make sure an event with the  correct information is created.
  - Make sure a review with the correct information is created.

#### Relevant tickets
Fixes #117 
Fixes #113 

#### Checklist
